aws-s3/aws-s3.go

```diff
@@ -20,20 +20,58 @@ type AWSS3Client struct {
 	bucket string
 }
 
+var AWS_BUCKET = "documents"
+
 func NewClient() interfaces.IDocUploader {
 	sess := session.Must(session.NewSession(&aws.Config{
 		// TODO: move to config
 		Endpoint:    aws.String("http://s3mock:9090"),
 		Region:      aws.String("us-west-2"),
 		Credentials: credentials.NewStaticCredentials("accessKey", "secretKey", ""),
+		// needed for local docker image, should work for aws s3 too
+		S3ForcePathStyle: aws.Bool(true),
```

Comment:
This change was made to configure the s3 client to use the `http://$host_url/bucket` format, to make integration with the docker image possible.